index.js

```diff
@@ -1,15 +1,27 @@
 'use strict';
 var isFinite = require('is-finite');
 
-module.exports = function (numbers) {
+module.exports = function (numbers, range) {
```

Comment:
I would call it `opts` instead of range. Maybe in the future, more arguments are added.